core/modules/node/config/views.view.recent_content.json

```diff
@@ -0,0 +1,559 @@
+{
+    "_config_name": "views.view.recent_content",
+    "name": "recent_content",
+    "description": "Recent content.",
+    "tag": "default",
+    "disabled": false,
+    "base_table": "node",
+    "human_name": "Recent content",
+    "core": "1.0",
+    "display": {
+        "default": {
+            "display_title": "Master",
+            "display_plugin": "default",
+            "display_options": {
+                "query": {
+                    "type": "views_query",
+                    "options": []
+                },
+                "access": {
+                    "type": "perm"
+                },
+                "cache": {
+                    "type": "none"
+                },
+                "exposed_form": {
+                    "type": "basic"
+                },
+                "pager": {
+                    "type": "some",
+                    "options": {
+                        "items_per_page": "3",
```

Comment:
Lets also retain the old number here:
```suggestion
                        "items_per_page": "10",
```
![image](https://github.com/backdrop/backdrop/assets/2423362/a32e4336-18a8-4d17-b19c-5eb98f9e4b0d)

Reply:
Done